_layouts/swagger.html

```diff
@@ -2,154 +2,21 @@
-                        {% for parameter in method[1].parameters %}
-                        <tr>
-                            <td>
-                                {% if parameter.required %}
-                                <span class="swagger-parameter-required">
-                                {% endif %}
-                                {{ parameter.name }}
-                                {% if parameter.required %}
-                                </span>
-                                {% endif %}
-                            </td>
-                            <td>{{ parameter.in }}</td>
-                            <td>{{ parameter.description }}</td>
-                            <td>
-                                {% if parameter.type %}
-                                {{ parameter.type | capitalize }}
-                                {% if parameter.items %}
-                                of {{ parameter.items.type | capitalize }}
-                                {% endif %}
-                                {% else %}
-                                String
-                                {% endif %}
-                            </td>
-                        </tr>
-                        {% endfor %}
-                    </tbody>
-                </table>
-            </div>
-            {% endif %}
-            {% if method[1].responses %}
-            <div class="swagger-response">
-                <h4>Responses</h4>
-                {% for response in method[1].responses %}
-                <h5>
-                    <span class="swagger-response-code">{{ response[0] }}</span>
-                    {{ response[1].description }}
-                </h5>
-                {% for content_type in swagger.produces %}
-                    {% if response[1].examples[content_type] %}
-                        {% assign example = response[1].examples[content_type] %}
-                        {% if content_type contains 'json' %}
-                            {% highlight json %}{{ example }}{% endhighlight %}
-                        {% elsif content_type contains 'xml' %}
-                            {% highlight xml %}{{ example }}{% endhighlight %}
-                        {% else %}
-                            {% highlight http %}{{ example }}{% endhighlight %}
-                        {% endif %}
-                    {% endif %}
-                {% endfor %}
-                {% endfor %}
-            </div>
-            {% endif %}
-        </div>
-    </div>
-    {% endfor %}
-</div>
-{% endfor %}
-
-<script type="text/javascript">
-// Helpers
-
-var slice = Array.prototype.slice;
-
-function $(expr, parent) {
-    return typeof expr === "string" ? (parent || document).querySelector(expr) : expr || null;
-}
+{% assign paths   = swagger.paths | sort_by_tag %}
 
-function $$(expr, parent) {
-    return slice.call((parent || document).querySelectorAll(expr));
-}
+{% for tag in paths %}
+  <h2 class="swagger-tag">{{ tag[0] }}</h2>
```

Comment:
I'd use `h3` here. You can style `.swagger-tag` to become bigger if you want. It would be nice to have an `id` attribute to link it via an anchor.

Reply:
Done